android/expoview/src/main/java/host/exp/exponent/taskManager/ExpoHeadlessAppLoader.java

```diff
@@ -14,6 +14,7 @@
 
 import host.exp.exponent.headless.InternalHeadlessAppLoader;
 
+@DoNotStrip
```

Comment:
In our proguard file, I see a lot of `-keep ... .DoNotStrip` rules. But I don't see a `-keep host.exp.exponent.**.DoNoStrip` rule, does this work as expected?

Reply:
In `packages/@unimodules/core/android/proguard-rules.pro`, I've added a rule that protects all classes which were annotated with `@DoNotStrip`. See https://github.com/expo/expo/blob/bd787410dab66332afdc47cf70c6a98b47bd27da/packages/%40unimodules/core/android/proguard-rules.pro#L10

I've also tested it building a minified version of the expo-client. The `host.exp.exponent.taskManager.ExpoHeadlessAppLoader` was in the bundle. So it seems to work.